MdeModulePkg/Include/Protocol/PcieDoeCapbility.h

```diff
@@ -0,0 +1,82 @@
+/** @file
+  PCIe DOE Capability structure definitions
+
+  Copyright (c) 2020, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef PCIE_DOE_CAPBILITY_H_
+#define PCIE_DOE_CAPBILITY_H_

```

Comment:
The MdePkg can contain the industry standard. There is nothing wrong to put PcieDoeCapbility.h to MdePkg/Include/IndustryStandard, please move it back.

(Sorry, I did not say that clearly before. My previous feedback is only for PCI_DOE protocol.)